source/includes/fundamentals/code-snippets/CollationCollectionExample.java

```diff
@@ -67,18 +79,24 @@ private static void findOneAndUpdateExample(MongoCollection<Document> collection
                         .collation(Collation.builder().locale("de@collation=phonebook").build())
                         .sort(Sorts.ascending("first_name"))
                         .returnDocument(ReturnDocument.AFTER));
+
+        // Prints the JSON representation of the updated document, if an update occurred           
```

Comment:
Suggestion:

I think it would be more grammatical to omit this comma.

```suggestion
        // Prints the JSON representation of the updated document if an update occurred           
```